phpstan.neon

```diff
@@ -7,7 +7,6 @@ parameters:
 	paths:
 		- .
 	excludePaths:
-		- lib/lib_phpQuery.php
```

Comment:
should CssXPath be excluded from phpstan scan?

Reply:
Exclusion doesn't seem necessary if it's not causing issues, but potentially fair point from a handful of wasted cpu cycles.

Reply:
For now, I would rather keep it (phpstan uses some caching) to be informed in case of regression